### DISCUSSION TITLE
Batch application support

### DISCUSSION THREAD

**Discussion by ocafebabe** [glowroot/glowroot#1251] [Ideas]
Hi,

This is more a question than a bug report but I was unable to find the answer in other bug reports and the wiki! I'm trying Glowroot with a standalone Java application (JPA/Hibernate) but I don't get any results under the "Transactions" screen!? I do have some results under the JVM screen though...

Do I have to activate or change a particular setting to make this work or it's simply unsupported?

Thanks,

Christian

**Comment by gregsting** [comment]
You'll probably need to play with instrumentation if it's a pure batch job, see example for spring batch: https://github.com/glowroot/glowroot/issues/385

**Comment by Never-lab** [comment]
Wishlist: first-class **batch** job monitoring UX. Use Background transactions + Instrumentation today. Enhancement.

.
